ui/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -127,7 +133,9 @@ export default class AccountDetailsModal extends Component {
               showExportPrivateKeyModal();
             }}
           >
-            {this.context.t('exportPrivateKey')}
+            {process.env.MULTICHAIN
+              ? this.context.t('showPrivateKey')
```

Comment:
The text here should have a different font as per Figma

Current:
![Screenshot 2023-04-25 at 5 24 58 PM](https://user-images.githubusercontent.com/39872794/234268706-bb86e096-9f72-495c-8cc3-6c758380c615.jpg)

Figma:



![Screenshot 2023-04-25 at 5 25 34 PM](https://user-images.githubusercontent.com/39872794/234268840-c225bc17-8d7c-4874-9d41-3fe3e4bfd76b.jpg)